Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -533,6 +620,9 @@ def main() -> None:
             email_address = demisto.args()["email_address"]
             result = add_risky_user_label(reco_client, email_address)
             return_results(result)
+        elif command == "reco-get-assets-user-has-access-to":
+            result = get_assets_by_owner(reco_client, demisto.args()["asset_owner"], demisto.args()["only_sensitive"])
```

Comment:
The `only_sensitive` argument is not mandatory, so if the user would not supply any value, this would fail as the argument wouldnt be in `demisto.args()`